.github/workflows/buid-push.yaml

```diff
@@ -0,0 +1,45 @@
+on:
+  pull_request:
+    types:
+      - closed
+    paths:
+      - "pkg/*/examples/go.sum"
+      - "pkg/*/examples/go.mod"
```

Comment:
I was thinking that we only build-push when dependency file changes are made, as if we build and push when an SDK change is made it will have no affect on the example images. So my idea was that we would encourage developers to make their SDK changes create a PR, then run the script to update dependency files, and then once the dependency files have been merged we can build and push the new images with the SDK changes.

Reply:
I meant as in if we make changes to only the SDK code, it will not have an affect on the images until we update the dependency files.